ROX-19203: Add info for product usage page (#64063)
openshift/openshift-docs: modules/viewing-product-usage.adoc

// Module included in the following assemblies:
//
// * operating/use-system-health-dashboard.adoc
:_content-type: PROCEDURE
[id="viewing-product-usage_{context}"]
= Viewing product usage data

[role="_abstract"]
{product-title-short} provides product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from {product-title-short} sensors. This information can be useful in estimating {product-title-short} consumption data for reporting.

[NOTE]
====
{ocp} provides its own usage reports; this information is intended for use with self-managed Kubernetes systems.
====

This page provides the following data:

* Currently secured CPU units: The number of virtual CPUs used by your {product-title-short} secured clusters as of the latest metrics received from sensors.
* Currently secured node count: The number of Kubernetes nodes secured by {product-title-short} as of the latest metrics received from sensors.

> * Maximum secured CPU units: The maximum number of virtual CPUs used by your {product-title-short} secured clusters for the time period as measured by the *Start date* and *End date*. This measurement only includes clusters that are still connected.

Author: I rewrote this a little based on the additional info you provided in slack which helped me understand better what this is measuring. Please lmk if it's OK or not.

* Maximum secured node count: The maximum number of Kubernetes nodes secured by {product-title-short} as as measured by the *Start date* and *End date*. This measurement only includes clusters that are still connected.

> * CPU units observation date: The date on which the data displayed on this page for the currently secured CPU units was collected.

Author: We follow IBM Style:

 data - noun
Use as a singular noun with a singular verb, even though data is the plural of datum.

* Node count observation date: The date on which the data displayed on this page for the currently secured node count was collected.

Data is collected every 5 minutes from the sensors, so there can be a short delay in displaying the current data. To view historical data, you must configure the *Start date* and *End date* and download the data file. The date range is inclusive and depends on your time zone.

[NOTE]
====
The data shown is not sent to Red Hat or displayed as Prometheus metrics.
====

.Procedure

. In the {product-title-short} portal, navigate to *Platform Configuration* -> *System Health*.
. Click *Show product usage*.
. In the *Start date* and *End date* fields, choose the dates for which you want to display data. This range is inclusive and depends on your time zone.

> . Optional: To download the data, click *Download CSV*.

Reviewer: ```suggestion
. Optional: To download the detailed data, click *Download CSV*.
```

// Module included in the following assemblies:
//
// * operating/use-system-health-dashboard.adoc
:_content-type: PROCEDURE
[id="viewing-product-usage_{context}"]
= Viewing product usage data

[role="_abstract"]

> {product-title-short} provides product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from {product-title-short} sensors. This information can be useful in estimating {product-title-short} consumption data for reporting.

Reviewer 2: ```suggestion
{product-title-short} provides product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from {product-title-short} sensors. This information can be useful to estimate {product-title-short} consumption data for reporting.
```

For more information on how CPU units are defined in Kubernetes, see link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu[CPU resource units].

[NOTE]
====
{ocp} provides its own usage reports; this information is intended for use with self-managed Kubernetes systems.
====

> This page provides the following data:

Reviewer 2: Which page?

Author: I meant this page in the UI, but am rewriting to avoid awkwardness.

* Currently secured CPU units: The number of Kubernetes CPU units used by your {product-title-short} secured clusters as of the latest metrics received from sensors.

> * Currently secured node count: The number of Kubernetes nodes secured by {product-title-short} as of the latest metrics received from sensors.

Reviewer 2: ```suggestion
* Currently secured node count: The number of Kubernetes nodes secured by {product-title-short} as of the latest metrics collection.
```

> * Maximum secured CPU units: The maximum number of CPU units used by your {product-title-short} secured clusters as measured hourly and aggregated for the time period of the *Start date* and *End date*.

Reviewer 2: ```suggestion
* Maximum secured CPU units: The maximum number of CPU units used by your {product-title-short} secured clusters, as measured hourly and aggregated for the time period defined by the *Start date* and *End date*.
```

> * Maximum secured node count: The maximum number of Kubernetes nodes secured by {product-title-short} as measured hourly and aggregated for the time period of the *Start date* and *End date*.

Reviewer 2: ```suggestion
* Maximum secured node count: The maximum number of Kubernetes nodes secured by {product-title-short}, as measured hourly and aggregated for the time period defined by the *Start date* and *End date*..
```

> * CPU units observation date: The date on which the data displayed on this page for the maximum secured CPU units was collected.

Reviewer 2: ```suggestion
* CPU units observation date: The date on which the maximum secured CPU units data was collected.
```

> * Node count observation date: The date on which the data displayed on this page for the maximum secured node count was collected.

Reviewer 2: ```suggestion
* Node count observation date: The date on which the maximum secured node count data was collected.
```

> Data is collected every 5 minutes from the sensors, so there can be a short delay in displaying the current data. To view historical data, you must configure the *Start date* and *End date* and download the data file. The date range is inclusive and depends on your time zone.

Reviewer 2: ```suggestion
The sensors collect data every 5 minutes, so there can be a short delay in displaying the current data. To view historical data, you must configure the *Start date* and *End date* and download the data file. The date range is inclusive and depends on your time zone.
```
